### PR TITLE
[Console] Remove deprecated code

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Component/Console/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+ * removed `TableStyle::setCrossingChar()` method in favor of `TableStyle::setDefaultCrossingChar()`
+ * removed `TableStyle::etHorizontalBorderChar()` method in favor of `TableStyle::setDefaultCrossingChars()`
```
Comment (prisis) [inline src/Symfony/Component/Console/CHANGELOG.md]: you missed the "s" her

Comment (chalasr) [reply src/Symfony/Component/Console/CHANGELOG.md]: thanks, fixed

**Comment by chalasr** [comment]
Failing `deps=high` build  should be fixed when merging.
Ready

**Comment by chalasr** [comment]
Now green thanks to #31734

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**